### ISSUE TITLE
Fails if jar files exist as dependencies

### ISSUE THREAD

**Issue by tobykurien** [ksoichiro/gradle-eclipse-aar-plugin#4]
I have some JAR file dependencies in my libs folder, so in my build.gradle I have:

`compile fileTree(dir: 'libs', includes: ['*.jar'])`

The plugin then fails with:

```
ERROR: Could not find dependency: -android-MyProject-libs-volley.jar
...

* What went wrong:
Execution failed for task ':generateEclipseDependencies'.
> Cannot get property 'file' on null object
```

**Comment by ksoichiro** [comment]
Thanks for your feedback! I'll check that soon.

**Comment by ksoichiro** [comment]
Fixed in the last commit.
Currently it's uploaded as `0.1.1-SNAPSHOT`.
Please set the snapshot repository URL to use this.

``` groovy
buildscript {
    repositories {
        mavenCentral()
        maven {
            url uri('https://oss.sonatype.org/content/repositories/snapshots/')
        }
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:1.0.0'
        classpath 'com.github.ksoichiro:gradle-eclipse-aar-plugin:0.1.1-SNAPSHOT'
    }
}
```

**Comment by tobykurien** [comment]
Works like a charm, thank you!

**Comment by ksoichiro** [comment]
Thanks for checking! I'll release it as `v0.1.1` soon.

**Comment by tobykurien** [comment]
There is one caveat here: since the jars I depend on are in the "libs" folder, and the plugin copies the other jars into the "libs" folder too, when using gradle to compile the project, it will see duplicate jar files everywhere. So I created a new "jarDependencies" folder to keep my needed jar files, so they are separate from the generated jar files. This works fine in both Eclipse and Gradle, although I did have to manually set it up in Eclipse and export those classes. Perhaps you might want to change the directory of the generated JAR files from "libs" to "jarDependencies" in the plugin (and add that to Eclipse classpath)? This would avoid the conflict and manual setting up.
